src/detect-byte.c

```diff
@@ -25,7 +25,7 @@
 #include "detect-byte.h"
 #include "detect-byte-extract.h"
 #include "detect-bytemath.h"
-
+#include "rust.h"
```

Comment:
Why is this additional include needed ? If nothing is added to the C file...

Reply:
Ok because `DetectByteMathData` is now defined in rust.h and no longer in detect-bytemath.h

Reply:
> Looks like commit [a7f62b9](https://github.com/OISF/suricata/commit/a7f62b9c78b5da69bf1b9a266a0462db1fd62836) can be squashed into commit [6411d30](https://github.com/OISF/suricata/commit/6411d30c592f5ca6fccfa7a4fcf0bfb5ed1c9dae) ;-)

I want to keep these commits separate -- one's for byte-math additions and the other is for formatting fixups.